charts/fin-pay-transparency/charts/backend-external/templates/deployment.yaml

```diff
@@ -42,6 +42,8 @@ spec:
               value: '1000'
             - name: LOG_LEVEL
               value: {{.Values.app.env.logLevel}}
+            - name: EXTERNAL_BACKEND_SERVER_URL
```

Comment:
Rename to BASE_URL

Reply:
Do you mean just `BASE_URL` or `EXTERNAL_BACKEND_SERVER_BASE_URL `? @sukanya-rath

Reply:
BASE_URL